src/components/TodoList.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div>
+    <ul>
+      <TodoItem
+          v-for="todo in getAllTodos" :key="todo.id"
```

Comment:
What's the rule for number elements in line? Why this questions is important at all?